modules/service/ot-json-service.js

```diff
@@ -0,0 +1,258 @@
+const Utilities = require('../Utilities');
+const { sha3_256 } = require('js-sha3');
+
+class OtJsonService {
```

Comment:
Can we rename this service to be something like OtJsonUtilities since all methods are static. 
All classes with static methods that we are using in project are named *Utilities, i think that we should do the same here. Also on all places we we are using const otJsonService = require (...) let's use OtJson.... with big first letter as we are using for all other classes with static methods.